index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Scorecard </title>
+    <h1> Bowling Scorecard </h1>
+    <script src="https://code.jquery.com/jquery-2.1.4.min.js"></script>
+  </head>
+  <body style="background-image: url('background_wood.png');">
+
+    <form id="round_one"> ROUND ONE
+      <section>
+        <label for="firstBowl"> First Bowl:</label><input type="text" id="usersFirstBowl"><br>
+        <label for="secondBowl"> Second Bowl:</label><input type="text" id="usersSecondBowl"><br>
+      </section>
+      <input type="submit"> -- </input>
+      <h2 id="result_round_one"></h2>
+    </form>
+
+    <form id="round_two"> ROUND TWO
+      <section>
+        <label for="firstBowl2"> First Bowl:</label><input type="text" id="usersFirstBowl"><br>
+        <label for="secondBowl2"> Second Bowl:</label><input type="text" id="usersSecondBowl"><br>
+      </section>
+      <input type="submit"> -- </input>
+      <h2 id="result_round_two"></h2>
+    </form>
+
+    <form id="round_three"> ROUND THREE
+      <section>
+        <label for="firstBowl"> First Bowl:</label><input type="text" id="usersFirstBowl"><br>
+        <label for="secondBowl"> Second Bowl:</label><input type="text" id="usersSecondBowl"><br>
+      </section>
+      <input type="submit"> -- </input>
+      <h2 id="result_round_three"></h2>
+    </form>
+
+    <form id="round_four"> ROUND FOUR
+      <section>
+        <label for="firstBowl"> First Bowl:</label><input type="text" id="usersFirstBowl"><br>
+        <label for="secondBowl"> Second Bowl:</label><input type="text" id="usersSecondBowl"><br>
+      </section>
+      <input type="submit"> -- </input>
+      <h2 id="result_round_four"></h2>
+    </form>
+
+    <form id="round_five"> ROUND FIVE
+      <section>
+        <label for="firstBowl"> First Bowl:</label><input type="text" id="usersFirstBowl"><br>
+        <label for="secondBowl"> Second Bowl:</label><input type="text" id="usersSecondBowl"><br>
+      </section>
+      <input type="submit"> -- </input>
+      <h2 id="result_round_five"></h2>
+    </form>
+    <section>
+      <h1 id="allScores"> Hello </h1>
+
+    </section>
+
+    <script src="src/scorecard.js"></script>
+    <script src="interface.js"></script>
```

Comment:
I've seen this in a bunch of places but not sure why these're put at the bottom, is it a style choice or is there a technical motivation?